Library/Formula/pam_u2f.rb

```diff
@@ -0,0 +1,41 @@
+class PamU2f < Formula
+  desc "Provides an easy way to use U2F-compliant authenticators with PAM."
+  homepage "https://developers.yubico.com/pam-u2f/"
+
+  url "https://github.com/Yubico/pam-u2f.git", :commit => "95fb0ec65475e8afa15aef1c14d1457d1f80fece"
+  version "1.0.2"
+  sha256 "d77dd150d65836c7b9a12d75d22159484663c11cea5739653bf31c44a3609522"
+
+  head "https://github.com/Yubico/pam-u2f.git"
+
+  depends_on "libu2f-host" => :build
+  depends_on "libu2f-server" => :build
+
+  # Required for `autoreconf --install`
+  depends_on "pkg-config" => :build
+  depends_on "libtool" => :build
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+
+  # Required for ./configure
+  depends_on "asciidoc" => :build
+
+  def install
+    system "autoreconf", "--install"
+
+    ENV["A2X"] = "/usr/local/bin/a2x --no-xmllint"
+    args = ["--prefix=#{prefix}", "--with-pam-dir=#{lib}/pam"]
+    system "./configure", *args
+    system "make", "install"
+  end
+
+  def caveats; <<-EOS.undent
+    To finish the installation, you will need to link the PAM module to /usr/lib/pam.
+      sudo ln -s #{prefix}/lib/pam/pam_u2f.so /usr/lib/pam
```

Comment:
This would just be `#{lib}/pam/pam_u2f.so` rather than `#{prefix}/lib/pam/pam_u2f.so`.

However, this symlink won't be possible on OS X 10.11 and Homebrew doesn't support people needing to install things into `/usr/lib`. You'll need a workaround. We'd also want `dylibs` rather than shared objects.